internal/config/config.go

```diff
@@ -217,7 +217,7 @@ func Get() *IngressConfig {
 			KafkaTrackerTopic:     options.GetString("KafkaTrackerTopic"),
 			KafkaDeliveryReports:  options.GetBool("KafkaDeliveryReports"),
 			KafkaAnnounceTopic:    options.GetString("KafakAnnounceTopic"),
-			ValidTopics:           strings.Split(options.GetString("ValidTopics"), ","),
+			ValidUploadTypes:      strings.Split(options.GetString("Valid_Upload_Types"), ","),
```

Comment:
Maybe it's just me as I'm not a fan of these types of naming schemes :D, but does this look better than the paradigm we already have above? IE: `ValidUploadTypes`?

Reply:
It looks better to me!  ;)  With the existing approach, you end up with env vars that look like `INGRESS_VALIDUPLOADTYPES` (or `INGRESS_VALIDTOPICS` with the old name).  This is not very readable in my opinion.  I like to split the env var names on word boundaries.